cluster/src/main/java/com/hartwig/pipeline/execution/dataproc/GoogleDataproc.java

```diff
@@ -109,9 +110,15 @@ private String jobIdAndClusterName(final RuntimeBucket runtimeBucket, final Spar
         return untrimmed.substring(0, Math.min(untrimmed.length(), 50));
     }
 
-    private boolean isDone(final String jobName, final RuntimeBucket runtimeBucket) {
-        List<StatusCheck.Status> finalStatuses = asList(StatusCheck.Status.SUCCESS, StatusCheck.Status.FAILED);
-        return finalStatuses.contains(getStatus(jobName, runtimeBucket));
+    private boolean isDone(final Job job, final String jobName, final RuntimeBucket runtimeBucket) {
+        LOGGER.info("Checking if job is done");
+        String state = job.getStatus().getState();
+        if (state.equals("DONE")) {
```

Comment:
Not sure this naming really correct. Isn't a"CANCELLED" or "ERROR" job "done"? Maybe isComplete is a better name?